### PR TITLE
Ensure that Pushover works for Emergency prio as well

### PR DESCRIPTION
**Pull request by onno-vos-dev**
While playing around with the Pushover notifications, I noticed that Emergency prio does not work as intended since the `expire` and `retry` parameters were not passed along. 

Code could be slightly simplified by always sending in the: `expire` and `retry` parameters regardless of the prio since it appears that Pushover is ignoring those unless the prio is set to Emergency but it felt cleaner to respect their api spec this way. 

Some notes:
- I did add new text strings to the code base but do not have translations in place for the other languages. Currently, the English strings would show up. Not sure how big of a deal this is? 
- I'm not convinced that Emergency should even be an allowed prio for SABnzbd. To me it feels crazy to allow SABnzbd to send out Emergency messages over Pushover. Different discussion and perhaps food for a different PR but I'm happy to hear some opinions on this

### CODE REVIEW

**Comment by Safihre** [comment]
So you would like to have this support? As you say, emergency might be a bit over the top..

**Comment by onno-vos-dev** [comment]
I don't know if Emergency should be supported. One could argue that for very large NAS servers, having an Emergency alarm when disks are full could be useful. I'm in doubt, perhaps a warning to use Emergency sparingly could be good enough? At the end of the day, the user would have to take responsibility for what level of alarms they want.

File: po/main/SABnzbd.pot [hunk 1]
```diff
@@ -3704,6 +3704,21 @@ msgstr ""
...
+
+#: sabnzbd/skintext.py [Pushover settings]
+msgid "Emergency expire"
+msgstr ""
+
+#: sabnzbd/skintext.py [Pushover settings]
+msgid "How many seconds your notification will continue to be retried"
+msgstr ""
+
 #: sabnzbd/skintext.py [Header for Pushbullet notification section]
```
[the remark this answers is not in the document]
Comment (onno-vos-dev) [reply po/main/SABnzbd.pot]: Running `python tools/extract_pot.py` gives me a huge diff though? Looks like it hasn't been run for a while and not been updated? What would you like me to do? Commit all those changes or? @Safihre

**Comment by Safihre** [comment]
Indeed, it's their own responsibility. 
Right now we are doing the Release Candidate for 2.3.0 so not merging it yet. But I have no objections really against it.

File: po/main/SABnzbd.pot (hunk 1 above)
[the remark this answers is not in the document]
Comment (Safihre) [reply po/main/SABnzbd.pot]: No need, I usually run it before a release-push. Separate from code commits.

[the remark this answers is not in the document]
Comment (onno-vos-dev) [reply po/main/SABnzbd.pot]: Squashed it out :+1:

**Pull request merged**